backend/tracim_backend/views/contents_api/comment_controller.py

```diff
@@ -102,10 +102,9 @@ def comment_translation(self, context, request: TracimRequest, hapic_data=None):
     @hapic.output_body(CommentsPageSchema())
     def content_comments(self, context, request: TracimRequest, hapic_data=None):
         """
-        Get all comments related to a content in asc order (first is the oldest)
+        Get every comments related to a content in asc order (first is the oldest)
```

Comment:
every comment (every + singular)

Reply:
Solved in [fd85e03](https://github.com/tracim/tracim/pull/5846/commits/fd85e03f05599f33b4ee4f2e616fd52f78605da7).